pkg/network/cluster_config.go

```diff
@@ -59,13 +59,14 @@ func ValidateClusterConfig(clusterConfig configv1.NetworkSpec) error {
 	}
 
 	if clusterConfig.NetworkType == "" {
-		return errors.Errorf("spec.networkType is required")
+		// the type is required: OpenShiftSDN (default) or OVNKubernetes
+		return errors.Errorf("spec.networkType is required, such as OpenShiftSDN or OVNKubernetes")
```

Comment:
we shouldn't mention OVNKubernetes in the error message if it's not even tech preview